lib/res/shared/auth_button.dart

```diff
@@ -2,18 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.7
-
 import 'package:auth_buttons/res/shared/auth_style.dart';
 import 'package:auth_buttons/res/shared/button_contents.dart';
 import 'package:auth_buttons/res/shared/shared_button.dart';
 import 'package:flutter/material.dart';
 
 class AuthButton extends StatelessWidget {
-  final Key key;
+  final Key? key;
 
   ///**[onPressed]** is a void function well be called when the button pressed.
-  final VoidCallback onPressed;
+  final VoidCallback? onPressed;
```

Comment:
like here the onPressed parameter should be required not nullable.

Reply:
Thanks, I missed that one!